beacon_node/beacon_chain/src/light_client_server_cache.rs

```diff
@@ -312,6 +370,96 @@ impl<T: BeaconChainTypes> LightClientServerCache<T> {
     pub fn get_latest_optimistic_update(&self) -> Option<LightClientOptimisticUpdate<T::EthSpec>> {
         self.latest_optimistic_update.read().clone()
     }
+
+    pub fn get_sync_committee_branch(
+        &self,
+        store: &BeaconStore<T>,
+        block_root: &Hash256,
+    ) -> Result<Option<FixedVector<Hash256, CurrentSyncCommitteeProofLen>>, BeaconChainError> {
+        let column = DBColumn::SyncCommitteeBranch;
+
+        if let Some(bytes) = store
+            .hot_db
+            .get_bytes(column.into(), &block_root.as_ssz_bytes())?
+        {
+            // TODO unwrap
+            let sync_committee_branch: FixedVector<Hash256, CurrentSyncCommitteeProofLen> =
+                FixedVector::from_ssz_bytes(&bytes).unwrap();
+
+            return Ok(Some(sync_committee_branch));
+        }
+
+        Ok(None)
+    }
+
+    pub fn get_sync_committee(
```

Comment:
These methods should be moved to hot_cold_store

Reply:
i've moved the relvant fn's to hot_cold_store